packages/@react-spectrum/datepicker/src/utils.ts

```diff
@@ -77,3 +77,16 @@ export function useFocusManagerRef(ref: FocusableRef<HTMLElement>) {
   }));
   return domRef;
 }
+
+export function useDateCharacterWidth(state) {
```

Comment:
is it actually a character width? or is it the width of the characters + some fudging number? is this actually `useDateSegmentWidth`?

Reply:
It's more like `useApproximateFormattedDateWidth`, but that is too long. I'll update it along with `characterCount` to something better.

Reply:
I'm fine with the hook name being long, it's private and i'd prefer the name be accurate